doc/rst/language/evolution.rst

```diff
@@ -3,18 +3,225 @@
 Chapel Evolution
 ================
 
-Like any language, Chapel has changed over time. This page is designed to
-capture significant language changes that have the possibility of breaking
-existing user codes or code samples from old presentations or papers that
-predated the changes.
+Like any language, Chapel has changed over time. This page is designed
+to describe significant language changes that have a high likelihood
+of breaking existing user codes or code samples from presentations or
+papers that predated the changes.
 
 Note that the compiler flag ``--warn-unstable`` is available and can be
 useful when migrating programs to the current version of the language.
-The purpose of this flag is to identify  portions of a program that use a
-language feature that has changed meaning.
+The purpose of this flag is to identify portions of a program that use a
+language feature which has changed meaning.  It also flags features that
+are considered unstable and may change in the future.
 
-version 1.21, September 2019
-----------------------------
+
+version 1.22, April 2020
+------------------------
+
+0- vs. 1-based Indexing
+***********************
+
+Version 1.22 makes a major change to Chapel with respect to indexing
+for cases that involve implicit indices.  Historically, Chapel has
+used 1-based indexing for such cases, where it now uses 0-based
+indexing.
+
+The major types that are affected by this change are tuples, strings,
```

Comment:
Does `list` deserve a mention here?

Reply:
Yeah, I think so.  I always seem to drop them on the floor mentally for some reason.